.github/workflows/pr_test.yml

```diff
@@ -46,4 +46,5 @@ jobs:
           pip install pytest
           pip install setuptools
           pip install -e ".[download]"
+          pip install -e ".[pvlib]"
```

Comment:
The installation with multiple extra groups could be combined into one. Please combine both lines into ``pip install -e ".[download,pvlib]"`` for efficient CI runs.